include/sway/config.h

```diff
@@ -98,6 +98,13 @@ struct seat_config {
 	list_t *attachments; // list of seat_attachment configs
 };
 
+enum config_dpms {
+    DPMS_IGNORE,
```

Comment:
Not all actually :)

Reply:
Well, yes... that is used too indicate that no DPMS configuration should be performed, if no dpms is set on the output command. And since enums defaults to the first entry, it is default.

Reply:
Oh, I was just talking about spaces vs tabs.